src/avfoundation.cs

```diff
@@ -3338,6 +3367,11 @@ interface AVQueuedSampleBufferRendering
 		[Abstract]
 		[Export ("stopRequestingMediaData")]
 		void StopRequestingMediaData ();
+
+		[TV (14,5), Watch (7,4), Mac (11,3), iOS (14,5)]
+		[Abstract]
```

Comment:
you can't add an `[Abstract]` member to a protocol
That's a breaking change since any existing code won't be implementing this (required) member

ObjC is less strict with `@required` / `@optional` than .net. Adding a `@required` is more a _suggestion_ than a requirement (and will work, at least, on older OS without that member).

We often bind those as `#if XAMCORE_4_0 [Abstract] #endif` and add it to the xtro `.ignore` file.